site/content/docs/developing/overrides/cdk.ja.md

```diff
@@ -0,0 +1,172 @@
+# CDK オーバーライド
+
+{% include 'overrides-intro.ja.md' %}
+
+## YAML パッチよりも CDK オーバーライドを使用するのはどの様な場合ですか？
+
+どちらのオプションも Copilot [Manifest](../../manifest/overview.ja.md) によって表面化されない機能にアクセスして設定するという"ガラスを壊す"仕組みです。
+
+プログラミング言語の表現力が必要な場合、[YAML パッチ](./yamlpatch.ja.md)よりも AWS Cloud Development Kit (CDK) オーバーライドをお勧めします。
+CDK を使えば、CloudFormation テンプレートに安全かつ強力な修正が行えます。
+
+## 始め方
+
+`copilot [noun] override` コマンドを実行すると、CDK　を使って CloudFormation テンプレートを拡張できます。
+例えば、`copilot svc override` コマンドにより、 Load Balanced Web Service のテンプレートを更新します。
+
+コマンドは、以下の様な構造で、`copilot/[name]/override` 配下に新しい CDK アプリケーションを作成します。
+```console
+.
+├── bin/
+│   └── override.ts
+├── .gitignore
+├── cdk.json
+├── package.json
+├── README.md
+├── stack.ts
+└── tsconfig.json
+```
+
+`stack.ts` を編集するとこから開始します。例えば、  `copilot svc override` を利用して、
+ECS サービスのプロパティをオーバーライドする場合、次の様な `stack.ts` ファイルが生成されので、修正をします。
+
+```typescript
+import * as cdk from 'aws-cdk-lib';
+import { aws_ecs as ecs } from 'aws-cdk-lib';
+
+export class TransformedStack extends cdk.Stack {
+    constructor (scope: cdk.App, id: string, props?: cdk.StackProps) {
+        super(scope, id, props);
+         this.template = new cdk.cloudformation_include.CfnInclude(this, 'Template', {
+            templateFile: path.join('.build', 'in.yaml'),
+        });
+        this.appName = template.getParameter('AppName').valueAsString;
+        this.envName = template.getParameter('EnvName').valueAsString;
+
+        this.transformService();
+    }
+ 
+    // TODO: implement me.
+    transformService() {
+      const service = this.template.getResource("Service") as ecs.CfnService;
+      throw new error("not implemented");
+    }
+}
+```
+
+## どの様な仕組みでしょうか？
+
+上記の `stack.ts` ファイルを見ればわかるとおり、Copilot は [cloudformation_include module](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.cloudformation_include-readme.html) を使用します。これは、CDK が提供している、変換を支援する為のライブラリです。
+このライブラリーは ["Import or migrate an existing AWS CloudFormation template"](https://docs.aws.amazon.com/cdk/v2/guide/use_cfn_template.html) ガイドで推奨されています。
```

Comment:
I think it would be better to translate 'library' to 'ライブラリ'. 
cf) [Wikipedia](https://ja.wikipedia.org/wiki/%E3%83%A9%E3%82%A4%E3%83%96%E3%83%A9%E3%83%AA)

Reply:
Understood. I change "ライブラリー" as "ライブラリ".